Services/Object/classes/class.ilObjectFactory.php

```diff
@@ -246,6 +246,8 @@ protected function includeClassIfNotExists(
         string $a_obj_type,
         ilObjectDefinition $objDefinition
     ) {
+        require_once("libs/composer/vendor/autoload.php");
```

Comment:
Please remove this.

Reply:
Of course. This was for testing. removed it.